metrics/multi_metrics.go

```diff
@@ -10,30 +12,22 @@ import "sync"
 // which can then be retrieved with Get(). This is for use with StressRelief. It
 // does not track histograms or counters, which are reset after each scrape.
 type MultiMetrics struct {
-	children []Metrics
-	values   map[string]float64
-	lock     sync.RWMutex
+	MList  *MetricsList `inject:"metricslist"`
```

Comment:
@kentquirk thinking through this problem some more I think that the root issue was that we weren't implementing the `Start` interface correctly.  I think that we can actually keep the original `GetMetricsImplementations` and `MultiMetrics` implementations as long as we implement the interface correct.

Reply:
I just played with this idea and confirmed it works.  We can safely remove all the `MetricsList` stuff.